eq-author-api/schema/resolvers/routing2/routingRule2/index.js

```diff
@@ -154,6 +155,7 @@ Resolvers.Mutation = {
     }, pages);
 
     const routing = page.routing;
+
```

Comment:
This file hasn't been modified and can be removed from commit

Reply:
idk why it's included or how to remove!

Reply:
Just remove the newlines on 149 and 158 and commit again.